### ISSUE TITLE
integrate google map

### ISSUE THREAD

**Issue by alincode** [alincode/digital-nomad-guide-to-taiwan#4]
to make easy to look for location.

**Comment by liuderchi** [comment]
Disclaimer: I am not a vue guy, following is just a quick survey 😄 

Had a quick google search found that [vue2-google-maps](https://www.npmjs.com/package/vue2-google-maps) may be an option:
  - support vue 2.x
  - rich [demos](http://xkjyeah.github.io/vue-google-maps/) for pure HTML/JS
  - 500+ downloads per day within last month

FYI

**Comment by alincode** [comment]
@liuderchi 
sounds good, I will try it. thank you.
what is your project now?

**Comment by liuderchi** [comment]
@alincode I'm still working on writing down ideas from my mind. 👨‍💻 

In short it's about making this [jupyter notebook UI plugin][article] working for NodeJS environment. 
(It supports python only right now)

![demo](https://blog.dominodatalab.com/wp-content/uploads/2016/05/sliderplot.gif)

[article]: https://blog.dominodatalab.com/interactive-dashboards-in-jupyter/ "article"

**Comment by alincode** [comment]
@liuderchi do you have repo for that now?

**Comment by liuderchi** [comment]
@alincode no. Perhaps I will start from forking the python version plugin:

[`jupyter-widgets/ipywidgets`](https://github.com/jupyter-widgets/ipywidgets)
